qa/L0_identity/model_repository/setup_identity_example.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
```

Comment:
This is the same as setup.sh

Reply:
Looking at the other example, it looks like the setup.sh is the duplicate.

Reply:
removed the duplicate